README.RDAP.md

```diff
@@ -34,45 +44,83 @@ Entity Primary Key can match multiple objects
+`type` non-conforming to the [RDAP spec](https://bitbucket.org/nroecg/nro-rdap-profile/raw/v1/nro-rdap-profile.txt)
+section 5.1.1, is not "home" or "work" as the RFC specifies. The `type` of this attribute is "abuse".
+
+In the previous paragraph `type` is considered as an element of the Jcard
+For example:
+````
+["adr",
+    {
+        "type":"home",
+        "label":"123 Maple Ave\nSuite 90001\nVancouver\nBC\n1239\n"
+    }
+]
+````
 
 Entity Search
 --------------------------
-Entity search on a handle is limited to returning 100 results.
+Entity search on a handle is limited to returning 100 results, so size and response time is not excessive.
+
+This is done as recommendation from the next RFC: https://datatracker.ietf.org/doc/rfc9083/ section 9. To conform with
+this spec a notification is added when the output is truncated.
 
 Domain Search
 --------------------------
-Domain search is restricted to only search for reverse delegations, and results are limited to 100.
+Domain search is restricted to only search for reverse delegations, there are no forward domains and results are
+limited to 100. So size and response time is not excessive.
+
+This is done as recommendation from the next RFC: https://datatracker.ietf.org/doc/rfc9083/ section 9. To conform with
+this spec a notification is added when the output is truncated.
 
 Netname may not match Whois
 ----------------------------
 The netname value returned by RDAP may not match what is returned by Whois.
 
 Nameserver queries always return Not Implemented
 -------------------------------------------------
-The RIPE database doesn't contain any forward domain objects, consequently a nameserver query will always return Not Implemented.
+The RIPE database doesn't contain any forward domain objects, consequently according to the RFC
+https://bitbucket.org/nroecg/nro-rdap-profile/raw/v1/nro-rdap-profile.txt section 6.3 "501 Not Implemented" will be
+returned.
 
 Only "mnt-by:" Maintainers are Listed as Registrants
 -----------------------------------------------------
-Only maintainers referenced in "mnt-by:" attributes will be listed as Registrants in responses.
+Only maintainers referenced in "mnt-by:" attributes will be listed as Registrants in responses. It is not relevant
+to include the rest of the mntners as they do not maintain the current object, they are not registrant of the object.
 
 Objects with "administrative" status are not returned
 -----------------------------------------------------
-If the prefix is either delegated but unallocated or only partially delegated to the RIPE region, then a 404 is returned. An object with "administrative" status is never returned.
+If the prefix is either delegated but unallocated or only partially delegated to the RIPE region,
+then a 404 is returned. An object with "administrative" status is never returned.
+
+Currently, IANA allocations are not present in the RIPE database, but just out-of-region placeholders. In the future
```

Comment:
Remove "In the future" sentence as it's making a commitment that is really uncertain (i.e. we don't have a plan to include it yet).